webpack/assets/javascripts/react_app/redux/actions/statistics/statistics.test.js

```diff
@@ -2,20 +2,26 @@ import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import immutable from 'seamless-immutable';
 
-import { requestData, onFailureActions } from './statistics.fixtures';
+import { failedRequestData, successRequestData, onFailureActions, onSuccessActions } from './statistics.fixtures';
 
 import * as actions from './index';
 
 const mockStore = configureMockStore([thunk]);
```

Comment:
do you need two stores? (mockStore and Store)

Reply:
`mockStore` is for store configuration with a middleware, where `store` is for initialization.